### ISSUE TITLE
Homepage Prototype

### ISSUE THREAD

**Issue by robinmdm** [monasobh/WebDocs-Assignment-Group-1#6]
We finished a prototype of the homepage based on your directives.
From the assignment guidelines, we're asking for your approval of the prototype. Let us know if everything is alright.

You can find our prototype in the folder assignment-2b-group6.

Thanks!

**Comment by bauscha** [comment]
Hello @robinmdm , 
sorry for the late response, unfortunatly our group has been away for the holidays. 
Thank you for the prototype. As far as we see it there is no CSS yet so we assume it will be added later, right? 

Without the CSS it's hard to give targeted feedback but here are a few ideas/suggestions for the following work we worked out as a group:
- Try generally to limit the with of content to about 1200px max-width so there is some empty space on full screen on the left and right side
- The Header above the navigation (including the logo) and the footer (including links to imprint and credits, Social Media icons and buttons) should be in the orange-tone of we submitted
- The navigation-bar on 100% width
- It would be great to use a font without serifs
- On the about page for the teacher-overview: Create "cards" for every teacher including the picture and name and separate it from the background by using a shadow
- Use the cards for each review as well


If you have any questions do not hesitate to ask!

**Comment by chrir0** [comment]
Hello, I am currently in the progress of adjusting and creating proper CSS and I have a question regarding the max-width. Do you want the header above the navigation also be limited to the max-width or just take the entire screen? Thank you in advance for letting me know!

I will let you know once I have finished my work and then update you so you can give us your final thoughts and small improvements we can still do.

**Comment by monasobh** [comment]
Hi, regarding your question @chrir0: please use the entire screen for the header above the navigation. 

Also, please provide us with one website (e.g. using github pages) for your prototyope.

**Comment by monasobh** [comment]
Dear developers, 
please be aware, that we did not receive any website from you to review the prototype. Please send us a link, once the prototype is finished. Thank you!

**Comment by chrir0** [comment]
That is my fault for the late update, please check it again tomorrow, by that time it should be complete. Apologies for the inconvenience and thank your for understanding

**Comment by robinmdm** [comment]
Here is the website link:

https://salsaswing.netlify.app/

Give us a feedback about what you think of this.

**Comment by chrir0** [comment]
Not yet, tomorrow please. Our deadline was extended slightly and I still need the time to finish it up. Thank you.

**Comment by chrir0** [comment]
I updated the CSS, we would appreciate some feedback especially regarding the usage of colors and fonts. Due to the deadline being very sure we won't be able to make big changes, but if there is any small issues let us know and we will try to fix them as soon as possible

**Comment by monasobh** [comment]
> I updated the CSS, we would appreciate some feedback especially regarding the usage of colors and fonts. Due to the deadline being very sure we won't be able to make big changes, but if there is any small issues let us know and we will try to fix them as soon as possible

Hi, are you sure this link https://salsaswing.netlify.app/ represents your current website? It seems as there is still CSS missing... 

<img width="1440" alt="Screenshot 2023-01-11 at 11 22 31" src="https://user-images.githubusercontent.com/49497676/211781717-4ec5bd74-dc44-4c7e-95a1-eaa957a70c94.png">

**Comment by chrir0** [comment]
I updated the code in the repository, my colleague is doing the hosting and I'm unsure. Hopefully he will fix it soon, sorry for the inconvenience. If you don't mind you could download the rep. that is available here and just open it in a browser as a preview in the meantime.

Edit: only 9 of the pages have CSS, but you should be able to tell by the names of the css files in addition to the imprint webpage, which didn't require an additional css file.

**Comment by bauscha** [comment]
Hello @chrir0 , thank you for the prototype. We checked the prototype on our local devices and found the following points to change:

- Footer -> "Imprint" and "Credit" -Links have a low contrast and are hard to read --> Change color to white or black
- Footer on mobile-format is not properly aligned (Background color cuts off)
- home.html and faq.html -> Footer is not at the bottom
- Social Media Icons are too big, please scale them down. Also if possible, make them one-colored (e.g. all Icons black)
- Navigation should be sticky when scrolling (only light grey part with links!)

- home.html -> Remove "Title" on the Textblock
- about-us -> Name of the teachers and their dance in the format like this example: "Chris (Salsa)" and center the names
- Schedule.html has no CSS
- courses.html -> Use the images of the detail pages in the overview
- news.html -> Images look stretched
- contact.html -> Only show Twitter, Instagram and Facebook
- credit.html has no CSS
- Align the navigation elements with the logo on the left side (li a: padding-left:0 and   .vl: padding-right
- Single Dance Sites (Irish Dance, Hip Hop, Charleston, Swing) -> CSS not applied
- Align the navigation and Logo in mobile format (<600px) to the center
- The Contact-Image (Map) on mobile (<600px) does not scale and gets cut off
- Check if you can align the stars on the same height in the Reviews for all cards
- Our URL is supposed to be "Salsa-swings", currently it's "Salsa-swing"

**Comment by chrir0** [comment]
Thank you for your in-depth feedback, I will try to fix the parts I am responsible for and open up another issue to let my colleague now which html issues are still open. As the deadline is soon there probably won't be another opportunity for a short review, however hopefully at least most of these can be fixed by then.

**Comment by monasobh** [comment]
Just a small hint for your submission tonight. I noticed that you renamed your index.html. According to this website from the course it is suggested to always keep index.html and not rename it. :) 

    [5 Mistakes Beginner Web Developers Make](https://www.freecodecamp.org/news/common-mistakes-beginning-web-development-students-make/) – and How to Fix Them, by Dave Gray

**Comment by robinmdm** [comment]
Dear Clients,

Thanks, @monasobh for pointing out the index.html issue. I fixed it, you can check it now, in the repository. ^_^

Thanks, @bauscha for your in-depth feedback. The domain name issue has been fixed. It is salsaswings now.

So, you can see the updated CSS version, on this website:

https://salsaswings.netlify.app/

@chrir0 , please fix the CSS issues.

We will enrich our website more. In the meantime, if you find anymore thing let us know!

**Comment by monasobh** [comment]
Dear developers,

thanks for your changes. I just saw that you centered the image on index.html and put the text below. I think your first version, image on the left and text on the right side, would be nice to have. Otherwise there is a lot of whitespace not used. Please adapt this. Thanks!

Other than that the page looks great. Thank you!

**Comment by robinmdm** [comment]
> 

Dear Clients,

index.html is back in its first edition, hope you have a pleasing website. ^_^

Still, if there is something you want, we would be happy to build it for you. Our ultimate goal is to exceed customer expectations and leave them with a sense of fulfillment.

Thanks! Regards!
